### ISSUE TITLE
OS Value on batch payments not being set to zero (6-2212)

### ISSUE THREAD

**Issue by steveblamey** [uzerpllp/uzerp#49]
There is a problem with pl batch payments - the OS-Value is not being set to £0 on batch payments. I thought we'd fixed this. There are 451 records in pltransactions where status ='P' and os_value <> 0 the first of which is dated 2014-02-05

So something changed around that time which hosed a previous fix.

**Comment by steveblamey** [comment]
Dave Easeman will update this in CVS against ticket 6-2212

**Comment by steveblamey** [comment]
This is now in base uzERP and CVS.

The code changes are as follows:-

modules/public_pages/erp/ledger/purchase_ledger/models/PLPayment.php

line 143 needs to be replaced with three lines as follows:-

take out 

```
#!php

             if (!$pltransaction->update($pltransaction->id, 'status', 'P'))
```
change to 

```
#!php

             if (!$pltransaction->update($pltransaction->id
                                         , array('status', 'os_value', 'twin_os_value', 'base_os_value')
                                         , array('P', '0.00', '0.00', '0.00')))

```

The version number at line 12 needs to be changed to 1.25.

**Comment by steveblamey** [comment]
fixes #49 (uzerp ticket 6-2212)
OS Value on batch payments not being set to zero


→ <<cset dd691071a9f6>>
